Description: Update system-deps to 6
Last-Update: 2022-10-03
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: rust-cairo-sys-rs-0.14.9/Cargo.toml
===================================================================
--- rust-cairo-sys-rs-0.14.9.orig/Cargo.toml
+++ rust-cairo-sys-rs-0.14.9/Cargo.toml
@@ -50,7 +50,7 @@ version = "2.16"
 features = ["xlib"]
 optional = true
 [build-dependencies.system-deps]
-version = "3"
+version = "6"
 
 [features]
 dox = ["x11/dox"]
